django_tables2/tables.py

```diff
@@ -411,6 +424,24 @@ def __init__(self, data, order_by=None, orderable=None, empty_text=None,
 
         self._counter = count()
 
+    def get_top_pinned_data(self):
+        '''
+        Return data for top pinned rows containing data for each row.
+        Iterable type like: queryset, list of dicts, list of objects.
+        Default return None. This method should be overridden in subclass of ~.Table
+        For None value, top pinned rows are not rendered.
```

Comment:
Did you check the output for this in the documentation? I'd like to see backticks around `None` to make it appear like code.

Please also use the more specific docstring syntax for the return value, to allow sphinx to render it like a return value. For example:
```
returns:
   `None` (default) no pinned rows at the top, iterable, data for pinned rows at the top.
```